File: app/src/main/java/at/bitfire/icsdroid/ui/ResourceInfo.java
/*
 * Copyright (c) Ricki Hirner (bitfire web engineering).
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the GNU Public License v3.0
 * which accompanies this distribution, and is available at
 * http://www.gnu.org/licenses/gpl.html
 */

package at.bitfire.icsdroid.ui;

import java.io.Serializable;
import java.net.URL;

public class ResourceInfo implements Serializable {

    URL url;
    boolean authRequired;
    String username;
    String password;

    Exception exception;

    int statusCode = -1;
    String statusMessage;

    String calendarName;
    int eventsFound = -1;

}

File: app/src/main/java/at/bitfire/icsdroid/ui/ResourceInfo.kt
/***************************************************************************************************
 * Copyright © All Contributors. See LICENSE and AUTHORS in the root directory for details.
 **************************************************************************************************/

package at.bitfire.icsdroid.ui

import android.net.Uri

data class ResourceInfo(

    var uri: Uri,

    var exception: Exception? = null,

    var calendarName: String? = null,
    var calendarColor: Int? = null,
    var eventsFound: Int = 0

)